services/contactsService.js

```diff
@@ -0,0 +1,54 @@
+const { Contact } = require("../db/contactModel");
+
+const getListContacts = async () => {
+  const contacts = await Contact.find({});
+  return contacts;
+};
+
+const getContactById = async (contactId) => {
+  const contact = await Contact.findById(contactId);
+  return contact;
+};
+
+const removeContact = async (contactId) => {
+  const contact = await Contact.findByIdAndRemove(contactId);
+  return contact;
+};
+
+const addContact = async ({ name, email, phone, favorite }) => {
+  const contact = new Contact({
+    name,
+    email,
+    phone,
+    favorite,
+  });
+  return await contact.save();
+};
+
+const updateContact = async (contactId, { name, email, phone }) => {
+  await Contact.findByIdAndUpdate(contactId, {
+    $set: {
```

Comment:
[options.overwrite=false] «Boolean» By default, if you don't include any update operators in doc, Mongoose will wrap doc in $set for you. This prevents you from accidentally overwriting the document. This option tells Mongoose to skip adding $set.
Link to docs: https://mongoosejs.com/docs/api.html#model_Model.update